sweetest/sweetest-api/src/test/java/com/mysugr/sweetest/BaseTest.kt

```diff
@@ -1,12 +1,12 @@
 package com.mysugr.sweetest
 
-import com.mysugr.sweetest.framework.environment.TestEnvironment
+import com.mysugr.sweetest.usecases.resetEnvironmentFully
 import org.junit.After
 
 open class BaseTest {
 
     @After
     open fun setUp() {
```

Comment:
A bit "unconventional" to call a `@After` function `setUp` 😅 
Is this for compatibility reasons?

Reply:
no, just was used to it that way 😁